### DISCUSSION TITLE
Action colors on a tags

### DISCUSSION THREAD

**Discussion by dotDennis** [incluud/accessible-astro-starter#96] [Q&A]
I'm unable to find the existing scss for the action tags used on for example page header/nav. I've tried looking through the node modules aswell, though its a headache to do so. Are these from tailwind, can i locally edit them, or is an overwrite necessary?

Thanks in advance ^^

**Comment by dotDennis** [comment]
Fresh day, fresh eyes. Found it in layouts ^^

**Comment by markteekman** [comment]
Hey @dotDennis! Wanted to reply to you on this but glad you've already found it 😊
